travelwise/frontend/templates/travel/create.html

```diff
@@ -13,10 +13,7 @@ <h2>Activities</h2>
         </div>
         <div class="container d-flex flex-column flex-grow-1">
           <div id="chat-messages" class="flex-grow-1 d-flex flex-column">
-            <div class="assistant">
```

Comment:
Lol the extra line

Reply:
Yeah, thought you had to use escape characters to do that but guess not lol